Update Set up your development doc for Medusa CLI section (#2866)
medusajs/medusa: docs/content/tutorial/0-set-up-your-development-environment.mdx

npm install @medusajs/medusa-cli -g
```

> You can check which version of Medusa you have by running the following command:

Reviewer: ```suggestion
You can check that Medusa was installed by running the following command:
```
